app/javascript/packs/AddNewRestPopup.jsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import $ from 'jquery';
+
+class AddNewRestPopup extends React.Component {
+
+    constructor() {
+        super();
+        this.state = {
+            name: "",
+            address: "",
+            tenbis: false,
+            max_delivery_time: 10,
+            cuisine: ""
+        };
+
+        this.updateChange = this.updateChange.bind(this);
+        this.newRestaurantSubmit = this.newRestaurantSubmit.bind(this);
+    }
+
+    updateChange(e) {
+        const event_id = e.target.id;
+        const event_value = e.target.value;
+
+        if (event_id === "restaurant_name") {
+            this.setState({name: event_value});
```

Comment:
Use `setState` that gets a function: https://medium.com/@baphemot/understanding-reactjs-setstate-a4640451865b